src/component/main-content.js

```diff
@@ -53,14 +53,34 @@ export default class MainContent extends React.Component {
     if (requestData.totalCount > 0) {
       SearchBar = <Search filterRequests={filterRequests} />;
     } else {
-      const browserElements = browsers.map((browser) => {
-        return <Browser browserName={browser.name} key={browser.name} openBrowser={openBrowser} />;
-      });
+      const browserElements = browsers
+        .sort((a, b) => {
+          if (a.name < b.name) return -1;
+          if (a.name > b.name) return 1;
+
+          if (a.version < b.version) return -1;
+          if (a.version > b.version) return 1;
+
+          return 0;
+        })
+        .map((browser) => {
+          if (browser.name === 'ie') {
+            browser.name = 'Internet Explorer';
+          } else if (browser.type === 'chrome' && /SxS/.test(browser.command)) {
+            browser.name = 'Chrome Canary';
+          } else if (browser.type === 'firefox' && /Developer Edition/i.test(browser.command)) {
```

Comment:
Using `browser.command` might not work if the user changed their installation path, but the design still provides a fallback to the version number, so the user can differentiate between the two installations even if their names are the same.